MyTestProject/src/Main.java

```diff
@@ -2,4 +2,9 @@ public class Main {
     public static void main(String[] args) {
         System.out.println("Hello world!");
     }
+
+    public void urgentTest() {
+        System.out.println("I am an urgent test");
+        System.out.println("This test is finished");
```

Comment:
Why is this line here? Is it necessary to tell all the world that the test is finished?